src/routes/log.ts

```diff
@@ -121,8 +121,8 @@ const prepareSQLFilters = ({
 }
 router.route('/api-stats').get(async (req: CustomRequest, res: Response) => {
   try {
-    const page = req.query.page || 0
-    const max = req.query.max || 5000
+    const page = Number(req.query.page) || 0
```

Comment:
Is the valid range of page & max?  Can they be passed in with negative values?  non-integer values?

Reply:
Good point I'll revisit, My guess is math will break and client doesn't get anything.